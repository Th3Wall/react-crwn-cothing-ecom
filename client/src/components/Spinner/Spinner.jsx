import { SpinnerOverlay, SpinnerContainer } from './Spinner.styles';

const Spinner = () => (
    <SpinnerOverlay>
        <SpinnerContainer />
    </SpinnerOverlay>
)

export default Spinner;