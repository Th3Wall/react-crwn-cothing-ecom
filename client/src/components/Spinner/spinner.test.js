import React from 'react';
import { shallow } from 'enzyme';
import Spinner from './Spinner';

describe('Spinner', () => {
    it('should display Spinner component', () => {
        const wrapper = shallow(<Spinner />);
        expect(wrapper).toMatchSnapshot();
    })
})