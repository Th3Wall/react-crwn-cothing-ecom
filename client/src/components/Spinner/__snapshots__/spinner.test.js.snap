// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should display Spinner component 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;
